core/ledger/kvledger/txmgmt/validation/validator.go

```diff
@@ -172,19 +172,21 @@ func (v *validator) validateReadSet(ns string, kvReads []*kvrwset.KVRead, update
 // i.e., it checks whether a key/version combination is already updated in the statedb (by an already committed block)
 // or in the updates (by a preceding valid transaction in the current block)
 func (v *validator) validateKVRead(ns string, kvRead *kvrwset.KVRead, updates *privacyenabledstate.PubUpdateBatch) (bool, error) {
+	readVersion := rwsetutil.NewVersion(kvRead.Version)
 	if updates.Exists(ns, kvRead.Key) {
+		logger.Warnf("Version mismatch for key [%s:%s]. Version in readSet %s but same key has been updated in a prior transaction in this block", ns, kvRead.Key, readVersion)
```

Comment:
Can we use the format that we have been using for the log to be as machine readable
```suggestion
logger.Warnf("Version mismatch. Key has been updated in a prior transaction in this block", "key", ns+":"+kvRead.Key, "Version in readSet", readVersion)
```

Reply:
Done